src/app/screens/Dashboard/screens/Home/components/CardsContainer/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { withSpinner } from '~components/Spinner';
+
+import Card from '../Card';
+import styles from './styles.module.scss';
+
+interface Props {
+  components: {
+    title: string,
+    html: string,
+    css: string
+  }[],
```

Comment:
We should move this to a `types` file or `interfaces`. This type will definitely change somewhere soon, so lets keep it more scalable  😄